### PR TITLE
Configuration pom.xml mail-notification

### PR DESCRIPTION
**Pull request by PotoracMihai**
I add in pom.xml dependecy required

### CODE REVIEW

File: mail-notification-service/pom.xml [hunk 1]
```diff
@@ -4,9 +4,91 @@
...
+        <url/>
+    </scm>
+    <properties>
+        <java.version>23</java.version>
+        <sonar.organization>jah3</sonar.organization>
+        <sonar.host.url>https://sonarcloud.io</sonar.host.url>
+    </properties>
+    <dependencies>
+        <dependency>
+            <groupId>org.projectlombok</groupId>
```
Comment (taragrammatonn) [inline mail-notification-service/pom.xml:38]: twice lombok

File: mail-notification-service/pom.xml [hunk 2]
```diff
@@ -4,9 +4,91 @@
...
+            <optional>true</optional>
+        </dependency>
+        <dependency>
+            <groupId>org.springframework.boot</groupId>
+            <artifactId>spring-boot-starter-test</artifactId>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.springframework.amqp</groupId>
+            <artifactId>spring-rabbit-test</artifactId>
```
Comment (taragrammatonn) [inline mail-notification-service/pom.xml:62]: use spring boot starter qmpq

**Review by taragrammatonn — CHANGES_REQUESTED** [review]

File: mail-notification-service/pom.xml (hunk 2 above)
Comment (PotoracMihai) [reply mail-notification-service/pom.xml:57]: El este acolo, mai jos

Comment (taragrammatonn) [reply mail-notification-service/pom.xml:57]: then remove this cause need to use starter ampq

Comment (PotoracMihai) [reply mail-notification-service/pom.xml]: I did

**Review by taragrammatonn — APPROVED** [review]

**Pull request merged**